Simplenote/Classes/InterlinkViewController.swift

```diff
@@ -57,11 +57,11 @@ extension InterlinkViewController {
         let (orientation, viewportSlice)    = calculateViewportSlice(around: anchor, in: viewport)
         let height                          = calculateHeight(viewport: viewportSlice)
         let topLocation                     = calculateTopLocation(for: height, around: anchor, orientation: orientation)
-        let leading                         = calculateLeadingLocation(around: anchor, in: viewportSlice)
+        let left                            = calculateLeftLocation(around: anchor, in: viewport)
 
         tableTopConstraint.constant         = topLocation
         tableHeightConstraint.constant      = height
-        tableLeadingConstraint.constant     = leading
+        tableLeftConstraint.constant        = left
```

Comment:
@jleandroperez this is where the conflicts were. I pieced it together based on what compiled.

I read the description of #1030, the PR that introduced the conflicting changes in `develop`, and went through the testing steps. It seemed to work okay, although I'm not sure what the expected behavior was 😳 .

interlink from top | interlink from bottom
--- | ---
![image](https://user-images.githubusercontent.com/1218433/99626806-1d922280-2a87-11eb-8f3c-fe1c153f7ee6.png) | ![image](https://user-images.githubusercontent.com/1218433/99626842-3569a680-2a87-11eb-9878-590616e7b9f1.png)

This was the diff, which you can see in #1033, too.

```diff
diff --cc Simplenote/Classes/InterlinkViewController.swift
index 736d96e8,9ec5c0fd..00000000
--- a/Simplenote/Classes/InterlinkViewController.swift
+++ b/Simplenote/Classes/InterlinkViewController.swift
@@@ -50,18 -50,17 +50,28 @@@ extension InterlinkViewController 
      /// Relocates the receiver so that it shows up **around** the specified Anchor Frame, in a given ViewPort.
      /// - Important: Both frames must be expressed in Window Coordinates. Capisce?
      ///
 -    func relocateInterface(around anchor: CGRect, in viewport: CGRect) {
 -        let anchorFrame = view.convert(anchor, from: nil)
 -        let editingRect = view.convert(viewport, from: nil)
 -
 +    func relocateInterface(around anchorInWindow: CGRect, in viewportInWindow: CGRect) {
 +        let anchor                          = view.convert(anchorInWindow, from: nil)
 +        let viewport                        = view.convert(viewportInWindow, from: nil)
 +
++<<<<<<< HEAD
 +        let (orientation, viewportSlice)    = calculateViewportSlice(around: anchor, in: viewport)
 +        let height                          = calculateHeight(viewport: viewportSlice)
 +        let topLocation                     = calculateTopLocation(for: height, around: anchor, orientation: orientation)
 +        let leading                         = calculateLeadingLocation(around: anchor, in: viewportSlice)
 +
 +        tableTopConstraint.constant         = topLocation
 +        tableHeightConstraint.constant      = height
 +        tableLeadingConstraint.constant     = leading
++=======
+         let targetHeight = calculateHeight()
+         let targetTop = calculateTopLocation(for: targetHeight, around: anchorFrame, in: editingRect)
+         let targetLeft = calculateLeftLocation(around: anchorFrame, in: editingRect)
+ 
+         tableTopConstraint.constant = targetTop
+         tableHeightConstraint.constant = targetHeight
+         tableLeftConstraint.constant = targetLeft
++>>>>>>> release/4.28
      }
  
      /// Adjusts the Interlink TableView by the specified offset
```

Let me know if this is right. Thanks.